### PR TITLE
[IA-3989] [AJ-805]: bump listener version for upload bug fix

### PR DESCRIPTION
**Pull request by rtitle**
https://broadworkbench.atlassian.net/browse/IA-3989
https://broadworkbench.atlassian.net/browse/AJ-805

Listener PR: https://github.com/DataBiosphere/terra-azure-relay-listeners/pull/25

Bumps listener version for notebooks and apps to fix a bug around uploading large files.

Tested manually in a notebook env -- will retest Leo in a BEE for notebooks and apps.

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green (you can re-run automation tests with a comment saying `jenkins retest`
- [ ] Run the automation tests multiple times in parallel to weed out instability if applicable via a comment saying `multi-test`
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,7 +23,7 @@ EXPOSE 5050
 ENV GIT_HASH $GIT_HASH
 ENV HELM_DEBUG 1
 # WARNING: If you are changing any versions here, update it in the reference.conf
-ENV TERRA_APP_SETUP_VERSION 0.0.4
+ENV TERRA_APP_SETUP_VERSION 0.0.5
```
Comment (rtitle) [inline Dockerfile]: Comes from https://github.com/DataBiosphere/terra-app/pull/27/

**Review by davidangb — APPROVED** [review]
:+1:, did not test it myself

**Review by LizBaldo — APPROVED** [review]
LGTM!

**Comment by rtitle** [comment]
Thanks - I'll just BEE-test it before merge

**Comment by rtitle** [comment]
Weird, the helm pull of the new setup chart failed in Jenkins -- but it's there in the repo. jenkins retest

File: Dockerfile [hunk 2]
```diff
@@ -56,6 +56,7 @@ RUN helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx && \
 # Leonardo will install the chart from local version.
 # We are also caching charts so they are not downloaded with every helm-install
 RUN cd /leonardo && \
+    helm search repo terra-app-setup-charts -l && \
```
Comment (rtitle) [inline Dockerfile]: Of course it worked as soon as I added this debug line… 🤦 , oh well

**Comment by rtitle** [comment]
fiab-start, jenkins retest

**Comment by rtitle** [comment]
jenkins retest

**Pull request merged**